.github/workflows/tests.yaml

```diff
@@ -18,6 +18,6 @@ jobs:
         run: git submodule update --init --recursive
 
       - name: Run tests
-        run: make test-forge
+        run: make test
```

Comment:
`test-forge¯ now is just an alias to `test`.
I'm changing this to make it match Goerli.